src/Browsershot.php

```diff
@@ -388,14 +388,20 @@ protected function callBrowser(array $command)
     {
         $setIncludePathCommand = "PATH={$this->includePath}";
 
-        $setNodePathCommand = "NODE_PATH=`{$this->nodeBinary} {$this->npmBinary} root -g`";
+        $nodeBinary = $this->nodeBinary ?: 'node';
+
+        if ($this->npmBinary) {
+            $setNodePathCommand = "NODE_PATH=`{$nodeBinary} {$this->npmBinary} root -g`";
+        } else {
```

Comment:
Let's avoid this `else` statement by introducing a new `getNodePathCommand` function. Use early returns in that new function.